services/QuillLMS/client/app/bundles/Staff/helpers/comprehension.tsx

```diff
@@ -207,6 +200,23 @@ export function getActivityPromptSetter({
   return updatePrompt;
 };
 
+export function getPromptForActivitySession(sessionData: any, conjunction: string) {
+  if(!sessionData) {
+    return null;
+  }
+  const { activitySession } = sessionData;
+  const { prompts } = activitySession;
+  if(prompts[conjunction]) {
+    const prompt = prompts[conjunction];
+    prompt.conjunction = conjunction;
+    return prompt;
+  }
+  return {
+    conjunction,
+    text: 'none'
```

Comment:
Why do we want to use the string `none` here? Wouldn't it work just as well to leave it undefined? I'm asking because I think it's easier to remember to check for the lack of presence of something rather than that it matches a special string that means "nothing". If we do need to use the string for some reason, we should constantize it in a shared file and import it in the places we use it.

Reply:
so the reason is because of the way we load the prompt in the `promptTable` component. While waiting for the data, we check for `!prompt` and render a loading spinner. There are cases where the activity session data will load, but  there may not be responses for all of the prompts, so I chose to do this to easily render in the UI like so:

```
  if(prompt && prompt.text === 'none') {
    const { conjunction, text } = prompt;
    return(
      <section className="prompt-table-container">
        <section className="attempts-section">
          <p className="attempts-label">{PROMPT_HEADER_LABELS[conjunction]}</p>
          <p className="attempts-value">{text}</p>
        </section>
      </section>
    );
  }
```